wire-compiler/src/test/java/com/squareup/wire/schema/WireRunTest.kt

```diff
@@ -558,6 +588,11 @@ class WireRunTest {
           |package squareup.polygons;
           |message Triangle {
           |  repeated double angles = 1;
+          |   enum Type {
```

Comment:
nittty as hell but there are 1 too many space on the indention

Reply:
good catch, will fix